src/deploy/functions/release/fabricator.ts

```diff
@@ -233,8 +233,16 @@ export class Fabricator {
           })
           .catch(rethrowAs(endpoint, "set invoker"));
       }
+    } else if (backend.isCallableTriggered(endpoint)) {
+      // Callable functions should always be public
+      const invoker = ["public"];
+      await this.executor
+        .run(async () => {
+          await gcf.setInvokerCreate(endpoint.project, backend.functionName(endpoint), invoker);
+        })
+        .catch(rethrowAs(endpoint, "set invoker"));
     } else if (backend.isTaskQueueTriggered(endpoint)) {
-      // Like HTTPS triggers, taskQueueTriggers have an invoker, but unlike HTTPS they don't default
+      // Like other triggers, taskQueueTriggers can have an invoker, but they don't default
```

Comment:
The previous comment is still true since only task queue triggers and http triggers can explicitly set invokers. I'd prefer if we revert this change.

Same comment for L315

Reply:
👍